schemas/can_tables.sql

```diff
@@ -9,208 +9,138 @@ INSERT INTO meta.actnow_intervention_types (id, name) VALUES
+INSERT INTO meta.actnow_timeseries_variables (id, name) VALUES
+(1, 'cumulative_deaths'),
+(2, 'cumulative_infected'),
+(3, 'cumulative_negative_tests'),
+(4, 'cumulative_positive_tests'),
+(5, 'hospital_bed_capacity'),
+(6, 'hospital_beds_required'),
+(7, 'icu_bed_capacity'),
+(8, 'icu_beds_in_use'),
+(9, 'rt_indicator'),
+(10, 'rt_indicator_ci_90'),
+(11, 'ventilator_capacity'),
+(12, 'ventilators_in_use');
+
+create table meta.actnow_actual_variables (
+    id serial primary key,
+    name text
+);
 
-- `1` corresponds to no intervention and an R0 of 3.7
-- `2` corresponds to weak intervention with social distancing only and an R0 of 1.7
-- `3` corresponds to strong intervention with shelter in place orders and R0 of 1.1 for 4 weeks, 1.0 for 4 weeks and 0.8 for four weeks
-- `4 corresponds to a dynamic measure of intervention based on observed policies (where availible). The R0 value is inferred from recent cases, hospitalizations, and deaths in each state';
-COMMENT ON COLUMN data.actnow_county_timeseries.fips is E'The 5 digit US fips code for the county.';
-COMMENT ON COLUMN data.actnow_county_timeseries.hospital_beds_required is E'The projected number of hopstial beds needed based on predicted number of active cases.';
-COMMENT ON COLUMN data.actnow_county_timeseries.hospital_bed_capacity is E'The total known number of hopsital beds.';
-COMMENT ON COLUMN data.actnow_county_timeseries.icu_beds_in_use is E'The number of projected ICU beds expected to be in use.';
-COMMENT ON COLUMN data.actnow_county_timeseries.icu_bed_capacity is E'The total number of known ICU beds potentially available.';
-COMMENT ON COLUMN data.actnow_county_timeseries.ventilators_in_use is E'The number of ventilators predicted to be in use';
-COMMENT ON COLUMN data.actnow_county_timeseries.ventilator_capacity is E'The total known number of ventilators.';
-COMMENT ON COLUMN data.actnow_county_timeseries.rt_indicator is E'The dynamic value of the re-infection rate.';
-COMMENT ON COLUMN data.actnow_county_timeseries.rt_indicator_ci_90 is E'The 90th percentile confidence interval for the upper bound on the value of rt';
-COMMENT ON COLUMN data.actnow_county_timeseries.cumulative_deaths is E'The proejcted number of cumulative COVID-19 deaths';
-COMMENT ON COLUMN data.actnow_county_timeseries.cumulative_infected is E'The projected, cumulative number of persons infected with COVID-19.';
-COMMENT ON COLUMN data.actnow_county_timeseries.cumulative_positive_tests is E'The projected, cumulative number of people who have tested positive for COVID-19.';
-COMMENT ON COLUMN data.actnow_county_timeseries.cumulative_negative_tests is E'The projected, cumulative number of people who have tested negative for COVID-19.';
-
-
-DROP TABLE IF EXISTS data.actnow_state_timeseries;
-CREATE TABLE data.actnow_state_timeseries (
-    "dt" DATE,
+INSERT INTO meta.actnow_actual_variables (id, name) VALUES
```

Comment:
Would it be particularly crazy to combine `meta.actnow_timeseries_variables` and `meta.actnow_actual_variables`? Lots of similar columns and might make the variable coding cleaner.

Reply:
they are totally different

one is input the other is outputs